simpleio.py

```diff
@@ -212,21 +231,23 @@ class DigitalIn:
       :param pull digitalio.Pull: pull configuration for the input
     """
 
-    def __init__(self, pin, **kwargs):
+    def __init__(self, pin: Pin, **kwargs: Any) -> None:
         self.iopin = digitalio.DigitalInOut(pin)
         self.iopin.switch_to_input(**kwargs)
 
     @property
-    def value(self):
+    def value(self) -> Any:
         """The digital logic level of the input pin."""
         return self.iopin.value
 
     @value.setter
-    def value(self, value):  # pylint: disable-msg=no-self-use, unused-argument
+    def value(self, value: Any) -> None:  # pylint: disable=no-self-use, unused-argument
```

Comment:
`bool` here as well I think for `value`.

Reply:
Done :)